admin_guide/install/docker_registry.adoc

```diff
@@ -358,6 +358,117 @@ Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
 
+== Exposing the Registry
+
+If you wish to expose your internal registry externally it is recommended that you run a
+link:#securing-the-registry[secure registry].  In order to expose the registry you must
+first have link:deploy_router.html[installed a router].
+
+. link:#deploy-registry[Deploy the registry].
++
+. link:#securing-the-registry[Secure the registry]
++
+. link:deploy_router.html[Deploy a router]
++
+. Create your passthrough route with `oc create -f <filename>.json`.  The passthrough route will
```

Comment:
Link this first mention of "passthrough route" to https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/routes.html#passthrough-termination since it may be new to the reader at this point.